src/tests/unit/tests/DetailsView/components/save-assessment-button.test.tsx

```diff
@@ -122,6 +109,12 @@ describe('SaveAssessmentButton', () => {
                     Times.atLeastOnce(),
                 );
             });
+            it('dialog is hidden (dismissed) when "got it" button is clicked', async () => {
+                const gotItButtonProps = getMockComponentClassPropsForCall(PrimaryButton);
+                gotItButtonProps.onClick();
+                const getProps = getMockComponentClassPropsForCall(Dialog);
```

Comment:
This is checking the props from the initial render.

Reply:
updated